hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
 # G400
 ![G400S](images/g400s.jpg) ![G400D](images/g400d.jpg)
+# Overview
+---
```

Comment:
Try not to use horizontal rules anywhere, the other docs don't use any so it doesn't fit the style.

Reply:
The horizontal rules were removed.